keyboards/aeboards/aegis/config.h

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2018 Jason Williams (Wilba)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4145 // "AE"
+#define PRODUCT_ID      0x0807 // 1800 -> 0x0708 -> 0x0807 ;-)
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    AEboards
+#define PRODUCT         Aegis
+#define DESCRIPTION     1800 Left Handed Keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 12
+#define MATRIX_COLS 9
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { F5, F6, E6, F7, D1, D0, D6, D4, B4, D7, B6, B5 }
+#define MATRIX_COL_PINS { C7, C6, B7, D2, D3, B3, B2, B1, B0 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
```

Comment:
This block is no longer necessary, as it's defined by default, and can be removed

If you want something other than both shifts, that's fine, but you may want to use `get_mods() & (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) ` instead, since we've found issues with using the keyboard_report struct.